### ISSUE TITLE
Error with `serf_client.members()`: `ValueError: invalid length of packed IP address string`

### ISSUE THREAD

**Issue by TechBK** [KushalP/serfclient-py#24]
I run serf with cmd:
```bash
serf agent -bind=192.168.15.129
```

My code:
```python
from serfclient.client import SerfClient


if __name__ == '__main__':
    serf_client = SerfClient()
    print(serf_client.stats())
    print(serf_client.members())
```

And traceback:
```bash
SerfResult<head={b'Error': b'', b'Seq': 1},body={b'runtime': {b'cpu_count': b'2', b'max_procs': b'2', b'arch': b'amd64', b'goroutines': b'30', b'os': b'linux', b'version': b'go1.5.2'}, b'serf': {b'left': b'0', b'encrypted': b'false', b'failed': b'0', b'query_queue': b'0', b'member_time': b'2', b'members': b'2', b'event_queue': b'0', b'query_time': b'1', b'event_time': b'1', b'intent_queue': b'0'}, b'event_handlers': {}, b'agent': {b'name': b'binh-lab-2'}, b'tags': {}}>


Traceback (most recent call last):
  File "/home/vcteam/.pycharm_helpers/pydev/pydevd.py", line 1531, in <module>
    globals = debugger.run(setup['file'], None, None, is_module)
  File "/home/vcteam/.pycharm_helpers/pydev/pydevd.py", line 938, in run
    pydev_imports.execfile(file, globals, locals)  # execute the script
  File "/home/vcteam/.pycharm_helpers/pydev/_pydev_imps/_pydev_execfile.py", line 18, in execfile
    exec(compile(contents+"\n", file, 'exec'), glob, loc)
  File "/home/vcteam/plugins/learn/learn_serfclient.py", line 7, in <module>
    print(serf_client.members())
  File "/usr/local/lib/python3.4/dist-packages/serfclient/client.py", line 50, in members
    return self.connection.call('members')
  File "/usr/local/lib/python3.4/dist-packages/serfclient/connection.py", line 86, in call
    for message in unpacker:
  File "msgpack/_unpacker.pyx", line 469, in msgpack._unpacker.Unpacker.__next__ (msgpack/_unpacker.cpp:469)
  File "msgpack/_unpacker.pyx", line 400, in msgpack._unpacker.Unpacker._unpack (msgpack/_unpacker.cpp:400)
  File "/usr/local/lib/python3.4/dist-packages/serfclient/connection.py", line 152, in _decode_addr_key
    ip_addr = socket.inet_ntop(socket.AF_INET6, obj_dict[key])
ValueError: invalid length of packed IP address string
```

**Comment by TechBK** [comment]
If i run serf with other cmd like: `serf agent` no error occur

**Comment by KushalP** [comment]
Closing as this was fixed by #25 and confirmed by @TechBK in https://github.com/KushalP/serfclient-py/pull/25#issuecomment-220763121
